.github/workflows/python-tests.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install flake8
-        python -m pip install -r requirements.txt
```

Comment:
You've deleted `requirements.txt` which included `pytest` and other dev packages. While you've added them here, there is no obvious way for a user to find which packages are required to install for testing.

There may be a better way, but the following is my understanding of how you should add these to your setup.cfg:

```
[options.extras_require]
dev =
    pytest>=2.9
    pytest-lazy-fixture
```

It is probably sensible to add `flake8` etc to these dev packages as well.

Reply:
You can install the dev packages alongside the module with, for example:
`pip install -e .[dev]`

Reply:
Done. Nice improvement, `python-test.yml `is cleaner